### PR TITLE
Math Magicians : Add Calculator design

### PR DESCRIPTION
**Pull request by AnicetFantomas**
In this pull request we performed :

- The use of components in React

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import './calculator.css';
+
+const Calculator = () => (
+  <section>
+    <div className="container">
+      <div className="display">
+        <div className="display-result">0</div>
+      </div>
```
Comment (codepantha) [inline src/components/calculator.js]: - [ ] Please make your Calculator component a class-based component 🙏

File: src/App.js [hunk 2]
```diff
@@ -1,28 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
   return (
```
Comment (codepantha) [inline src/App.js]: - [ ] Please make your App component a  class-based component 🙏

File: src/components/calculator.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      <div className="all-buttons">
+        <button type="button" className="button clear-all">AC</button>
+        <button type="button" className="button last-entity-clear">+/-</button>
+        <button type="button" className="button operation">%</button>
+        <button type="button" className="button operation">/</button>
```
Comment (codepantha) [inline src/components/calculator.js]: - [ ] Please model your styling according to the provided template 👇 
### Your Design
image
### Given Design
image

**Review by codepantha — CHANGES_REQUESTED** [review]
Hi @AnicetFantomas 👋, 

## STATUS: REQUESTED CHANGES ♻️ 

Good job so far! 👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Consider:

- [ ] Creating a reusable `<Button>` component for your Calculator. This will make your project better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShoiraTa — APPROVED** [review]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @AnicetFantomas , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**